service/routes.py

```diff
@@ -131,6 +131,26 @@ def get_items(wishlist_id, item_id):
     app.logger.info("Returning item: %s", item.name)
     return make_response(jsonify(item.serialize()), status.HTTP_200_OK)
 
+######################################################################
+# DELETE A ITEM
+######################################################################
+@app.route("/wishlists/<int:wishlist_id>/items/<int:item_id>", methods=["DELETE"])
+def delete_items(wishlist_id, item_id):
+    """
+    Delete a Item
+
+    This endpoint will delete a Item based the id specified in the path
+    """
+    app.logger.info("Request to delete item with id: %s", item_id)
+    item = Item.get_by_wishlist_id_and_item_id(wishlist_id, item_id)
+    if not item:
+        raise NotFound("Item with id '{}' was not found.".format(item_id))
```

Comment:
isn't ```DELETE``` idempotent?  i think if the wishlist exists, we always return ```200``` even if the item doesn't exist anymore

Reply:
we should return ```400``` if the wishlist doesn't exist, and then ```200``` if the item_id exists or does not exist